.github/workflows/pr-build-connector-template.yml

```diff
@@ -0,0 +1,51 @@
+name: PR Build (Connector)
+
+on:
+  workflow_call:
+    inputs:
+      additional-build-flags:
+        required: false
+        type: string
+        default: ""
+      additional-test-flags:
+        required: false
+        type: string
+        default: ""
+
+
+jobs:
+    ubuntu-build:
+        name: Build on Ubuntu
```

Comment:
We can change the job to `Build` since we don't have separate builds with this.

Reply:
updated